scripts/results_generation/generate_reproductions.py

```diff
@@ -33,12 +33,19 @@ def main():
         "brush_picking_up",
         "rail_cleaning",
     ]
-    base_folder = Path(__file__).parent.parent.parent.joinpath("data/maintenance_tasks")
+    # read noise standard deviation from input argument
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--std_dev", type=float, default=1, help='noise standard deviation')
+    parser.add_argument("--data_folder", type=str, default="maintenance_noise", help='data folder name')
+    parser.add_argument("--nb_repro", type=int, default=30, help='number of reproduction')
+    args = parser.parse_args()
+    std_dev = args.std_dev
+    assert std_dev > 0, "std_dev has to be positive"
```

Comment:
raise is better than assert because assert doesn't throw in optimized mode :)

Reply:
Fixed :)